kornia/filters/sobel.py

```diff
@@ -12,6 +12,15 @@ def _get_sobel_kernel_3x3() -> torch.Tensor:
     ])
 
 
+def _get_separate_kernel_3x3() -> torch.Tensor:
```

Comment:
should we place this to a kornia.filters.kernels.py file ?

Reply:
We probably should, but then I would move there all the existing - gaussian, box, sobel, which is  a separate refactoring effort

Reply:
I`ll do an issue for that

Reply:
Changed my mind - done in current PR